devlab/Vagrantfile

```diff
@@ -1,39 +1,41 @@
 # -*- mode: ruby -*-
 # vi: set ft=ruby ts=2 sw=2 et sua= inex= :
 
-public_key_path = "#{ENV["HOME"]}/.ssh/id_rsa.pub"
+config_file = "config.ini"
+options = {}
+File.foreach(config_file) { |line|
+  option, value = line.split("=")
+  option = option.strip()
+  value = value.strip()
+  options[option] = value
+}
+
 user = %x(id -un).strip()
 uid = %x(id -u #{user}).strip()
 
-# This is relative to user's $HOME, will be replaced with ${HOME}/${cloudferry_path}
-cloudferry_path = "Documents"
-
-
 nodes = {
   "grizzly" => {
     "box" => "openstack-user/precise-grizzly",
-    "ip" => "192.168.1.2",
+    "ip" => "#{options['src_host']}",
     "memory" => 4096,
     "role" => "openstack"
   },
   "icehouse" => {
     "box" => "openstack-user/precise-icehouse",
-    "ip" => "192.168.1.3",
+    "ip" => "#{options['dst_host']}",
     "memory" => 4096,
     "role" => "openstack"
   },
   "cloudferry" => {
     "box" => "hashicorp/precise64",
-    "ip" => "192.168.1.4",
+    "ip" => "#{options['snapshot_host']}",
```

Comment:
You already have `grizzly_ip`, `icehouse_ip` and `cloudferry_ip` in `config.ini`, let's use those, since they're way more readable